test/test_cuda_mapper.cc

```diff
@@ -758,7 +757,6 @@ def fun(float(N, M) A, float(N, M) B) -> (C,D) {
   auto tiledBand =
       ScheduleTree::makeScheduleTree(*scop->tileOuterBand(tiling.view));
 
-  using detail::ScheduleTreeElemBand;
```

Comment:
The removal of the `detail::` qualification in this file could have been done up front.